modules/mailer/daemon.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2014 The Gogs Authors. All rights reserved.
```

Comment:
Don't need this line for new files (I think? someone correct me if I'm wrong)

Ditto elsewhere